aimmo-game/simulation/world_map.py

```diff
@@ -68,10 +68,23 @@ class WorldMap(object):
     """
 
     def __init__(self, grid):
-        self.grid = grid
+        self._grid = grid
+
+    @classmethod
+    def generate_empty_map(cls, height, width):
+        start_x = -width / 2 + 1
+        end_x = width / 2 + 1
+        start_y = -height / 2 + 1
+        end_y = height / 2 + 1
```

Comment:
I think it was mentioned these implicitly floor when you do (width / 2).
Maybe it is because I am unfamiliar with python , but explicit flooring would be preferable)